gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/sanitizer/HtmlSanitizer.java

```diff
@@ -33,7 +33,12 @@ public final class HtmlSanitizer {
 
     private static final Parser mdParser = Parser.builder(new MutableDataSet()).build();
     private static final HtmlRenderer htmlRenderer = HtmlRenderer
-        .builder(new MutableDataSet().set(HtmlRenderer.SUPPRESSED_LINKS, ""))
+        .builder(
```

Comment:
A specific unit test would help to understand the use case.
Maybe one case passing the control and another rejected because it is invalid?

Reply:
actually I had to rewrite it with the comment block visitor to validate html like this: 
<!---><img src=x onerror=alert(1) />
the comment escape or comment suppressed was not enough to prevent such xss attack (the only difference is there is no new line after a invalid comment 

Please review once again